start_esp/start_esp.py

```diff
@@ -657,6 +664,15 @@ def make_argparser():
         Only works when --cors_preset is in use. Configures the CORS header
         Access-Control-Expose-Headers. Defaults to allow common response headers.
         ''')
+    parser.add_argument('--location',
+        help='''
+        One of acceptable zones, could not be any strings. Need to talk to
+        Chemist team to find out the acceptable zones.
+        If specified, send to ESP for location field and it will NOT be
+        overridden by Metadata server
+        ''')
```

Comment:
This is for us to talk to Chemist team.   Don't show in in the help.

Reply:
Removed location related changes